### DISCUSSION TITLE
Way to make password pop-up window show in task bar?

### DISCUSSION THREAD

**Discussion by Omni-Consumer-Products** [sumatrapdfreader/sumatrapdf#2378] [Q&A]
Hello,

Does anyone know of a way to force the password pop-up window that appears when you double click a password protected PDF to show up in the task bar?   After you enter the password the Sumatra main window shows up in the taskbar, but if you click on other things before entering the password, it's easy to lose the password window under other windows and somewhat of a pain to find it.  

TIA!

PS - thank you for making/maintaining SumatraPDF!  I uninstalled Acrobat Reader years ago after finding Sumatra and loving it.  I thought that I'd reinstall it if I ever needed it.   It's been at least 2 years, no Acrobat Reader installed :)

**Comment by GitHubRulesOK** [comment]
There should be no need once an item such as SumatraPDF on the task bar is select it should be in focus UNLESS you have an ON TOP app at the point you click on SumatraPDF the password dialog should be in-front
![image](https://user-images.githubusercontent.com/12754913/155902042-11541b79-4045-48ca-888a-b9d3aee59475.png)


OK  I see what you mean that on first occasion it is not running thus can be masked

![image](https://user-images.githubusercontent.com/12754913/155902167-c7f24704-5a3e-43f0-b58c-f76d979e2153.png)

@kjk any way that dialog can be forced on top whilst required.

**Comment by kjk** [comment]
I've added call to `BringWindowToTop` for password dialog but I'm not sure if that'll solve the issue

**Reply by Omni-Consumer-Products** [reply]
Thanks guys!   I'll try it out when the next update arrives.
